src/sdk/models/policies.ts

```diff
@@ -63,9 +63,43 @@ export interface PolicyItem extends TransferPlatformItem {
 	group?: PolicyTrackingGroup;
 }
 
+export enum PolicyAuditEvent {
```

Comment:
Rather than having this type here, I think that we should stick to a mapping similar to what is in [core](https://github.com/portalarchitects/skysync-vnext/blob/develop/Hosting/web/app/pages/Jobs/View/ActivityLog/util.tsx#L17).

Reply:
Yeah that works, once you showed me that mapping this typing became obsolete. I'll close this PR